### PR TITLE
Story3 eraser

### PR DESCRIPTION
**Pull request by katestarks**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -45,6 +45,71 @@ main {
...
+    margin-top: 30px;
+    background-color: #b2b2b2;
+    background-image: url("../IMG/eraserBlack.png");
+    border: 4px solid #b2b2b2;
+    box-shadow: 0 0 10px 10px rgba(178, 178, 178, 0.3);
+    -o-transition: .5s;
+    -ms-transition: .5s;
+    -moz-transition: .5s;
+    -webkit-transition: .5s;
+    cursor: pointer;
```
Comment (charliecog) [inline css/styles.css]: This is repeated a bunch of times, cant you just put it on main?

File: css/styles.css [hunk 2]
```diff
@@ -45,6 +45,71 @@ main {
...
+    -moz-transition:.5s;
+    -webkit-transition:.5s;
+    cursor: pointer;
+}
+
+.eraser {
+    margin-top: 30px;
+    border-color: #b2b2b2;
+    background-image: url("../IMG/eraserGrey.png");
+    box-shadow: 0 0 5px 5px rgba(178,178,178, 0.3), inset 0 0 5px 5px rgba(178,178,178, 0.3);
```
Comment (charliecog) [inline css/styles.css:78]: this box shadow and the transition is repeated twice, make into a class?

File: css/styles.css [hunk 3]
```diff
@@ -45,6 +45,71 @@ main {
...
+    border: 8px solid white;
+    border-radius: 40px;
+    margin: 0 auto 15px auto;
+    background-size: cover;
+}
+
+.white {
+    border-color: #FFFFFF;
+    background-image: url("../IMG/brushWhite.png");
+    box-shadow: 0 0 5px 5px rgba(255,255,255, 0.3), inset 0 0 5px 5px rgba(255,255,255, 0.3);
```
Comment (charliecog) [inline css/styles.css:60]: this box shadow and the transition is repeated twice, make into a class?

**Review by charliecog — COMMENTED** [review]
Doesn't work in ie10?

File: css/styles.css (hunk 1 above)
Comment (HazDuck) [reply css/styles.css]: done - cursor on the button class

File: css/styles.css (hunk 2 above)
Comment (HazDuck) [reply css/styles.css:78]: slightly different box shadows for slightly different buttons

File: css/styles.css (hunk 3 above)
Comment (HazDuck) [reply css/styles.css:60]: slightly different box shadows for slightly different buttons

**Review by charliecog — APPROVED** [review]
nice code!

**Pull request merged**